components/form/CheckboxGroup.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useFormContext } from 'react-hook-form';
+import PropTypes from 'prop-types';
+import { Label, ErrorLabel } from '.';
+
+export function Checkbox({ containerClassName, inputClassName, text, name, value }) {
+  const { register } = useFormContext();
+  return (
+    <>
+      <div className={`flex items-center ${containerClassName}`}>
+        <input
+          id={`checkbox-${name}-${value.toString()}`}
+          className={`w-4 h-4 text-amcovad-primary-400 border-gray-200 rounded focus:ring-amcovad-primary-300 ${inputClassName} `}
+          name={name}
+          type="checkbox"
+          {...register(name)}
+          value={value}
+        />
+        <Label name={name} htmlFor={`checkbox-${name}-${value.toString()}`} text={text} />
+      </div>
+    </>
+  );
+}
+export function CheckboxGroup({ name, options }) {
```

Comment:
Taking another look at this, I think we need to declare options in the propTypes and defaultProps not `label` and `value` 🤦🏽